cmd/imageplugin/main.go

```diff
@@ -29,7 +29,7 @@ func init() {
 
 var (
 	endpoint   = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	driverName = flag.String("drivername", "csi-image", "name of the driver")
+	driverName = flag.String("drivername", "image.csi.k8s.io", "name of the driver")
```

Comment:
does the driver name need to be configurable?

Reply:
It was in other drivers. I just kept it. If we think it should be hard coded, we can change it back.